purchase_report_custom_hls/models/res_partner.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class Partner(models.Model):
+    _inherit = "res.partner"
+
+    customer_incoterm_place = fields.Char(
+        "Default Incoterm Place",
+        help="The default incoterm place for purchase report.",
+    )
```

Comment:
Is there a requirement to keep the default incoterm place in partner?  Why customer?

Reply:
Maybe not use, Thank you.